packages/connect/package.json

```diff
@@ -4,12 +4,12 @@
   "description": "Substrate-connect to Smoldot clients. Using either substrate extension with predefined clients or an internal smoldot client based on chainSpecs provided.",
   "author": "Parity Team <admin@parity.io>",
   "license": "GPL-3.0-only",
+  "type": "module",
```

Comment:
did the position of this change for a reason?

Reply:
Absolutely no reason :)